packages/frontend/components/composed/issue/RelatedIssues/RelatedIssueBlock.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import { NamedLink, CoverImage, PrecisionDate } from "components/atomic";
+import { RelatedIssueBlockFragment$key } from "@/relay/RelatedIssueBlockFragment.graphql";
+import * as Styled from "./RelatedIssues.styles";
+import { getRouteByEntityType } from "helpers";
+
+export default function RelatedIssueBlock({ data }: Props) {
+  const issue = useMaybeFragment(fragment, data);
+
+  return issue && issue.slug ? (
+    <li key={issue.slug}>
+      <Styled.ItemCover>
+        <NamedLink
+          route={getRouteByEntityType(issue.__typename) || "home"}
+          routeParams={{ slug: issue.slug }}
+          passHref
+        >
+          <Styled.ItemCoverLink>
+            <CoverImage
+              title={issue.title}
+              id={issue.id}
+              data={issue.thumbnail}
+              maxWidth={263}
+              maxHeight={280}
+              usePlaceholder
+            />
+          </Styled.ItemCoverLink>
+        </NamedLink>
+      </Styled.ItemCover>
+      <Styled.ItemText>
+        <h4>
+          <NamedLink
+            route={getRouteByEntityType(issue.__typename) || "home"}
+            routeParams={{ slug: issue.slug }}
+            passHref
+          >
+            <a>{issue.title}</a>
+          </NamedLink>
+        </h4>
+        <div className="t-copy-sm a-color-lighter">
+          {issue.volume && <p>{issue.volume.title}</p>}
+          {issue.journal && <p>{issue.journal.title}</p>}
```

Comment:
I threw this in in case there are linked issues not from the same journal, but can take out if all links will be from the same journal (or if just not helpful).

Reply:
That's a good question - I think issues would usually be from the same journal, so we wouldn't need the journal name. But my assumption might be wrong... @zdavis , what do you think?

Reply:
Pretty hard to say. Might have to see how this plays out in practice.